cmd/thanos/compact.go

```diff
@@ -49,8 +49,8 @@ func (cs compactionSet) levels(maxLevel int) ([]int64, error) {
 		return nil, errors.Errorf("level is bigger then default set of %d", len(cs))
 	}
 
-	levels := make([]int64, maxLevel)
-	for i, c := range cs[:maxLevel] {
+	levels := make([]int64, maxLevel+1)
+	for i, c := range cs[:maxLevel+1] {
```

Comment:
Can we somehow add the info that we you start from `0` by chossing max_level not from 1 in flag description?

Reply:
actually, our flag is hidden, there were the following info https://github.com/improbable-eng/thanos/blob/master/cmd/thanos/compact.go#L92 https://github.com/improbable-eng/thanos/blob/master/cmd/thanos/compact.go#L41

that moment, when hidden flag should be visible to avoid ambiguity